resilience4j-spring/src/main/java/io/github/resilience4j/circuitbreaker/configure/ReactorDecoratorExt.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2019 Mahmoud Romeh
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.github.resilience4j.circuitbreaker.configure;
+
+import io.github.resilience4j.circuitbreaker.CircuitBreaker;
+import io.github.resilience4j.reactor.circuitbreaker.operator.CircuitBreakerOperator;
+import io.vavr.CheckedFunction0;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+/**
+ * the Reactor breaker logic support for the spring AOP Conditional on Reactor class existence on
+ * spring class loader
+ */
+public class ReactorDecoratorExt implements CircuitBreakerDecoratorExt {
```

Comment:
Why did you remove `CircuitBreaker` prefix? I think this should be `ReactorCircuitBreakerDecoratorExt`

Reply:
I can revert it